python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
     num = 0
     for v in vowels:
         num += text.lower().count(v)        
     return num
 
 def num_consonants(text):
-    vowels = "aeiou"
+    vowels = "aeiouy"
+    num = 0
     for letter in text:
         if letter not in vowels:
-            print("consonant", letter)
+           num += text.lower().count(letter)
+    return num
     
 text = str(input("Enter a sentence: "))
```

Comment:
the pr mentions test code that you're using, i'm not sure if you're using this correctly? could we chat in a few?